tests/swarm-deploy/test_service_restart.py

```diff
@@ -70,17 +73,29 @@ def deployed_simcore_stack(
     return core_stack_services
 
 
+SERVICES_AND_EXIT_CODES = [
+    ("webserver", 0),
+    ("static-webserver", 15),
+    ("storage", 0),
+    ("catalog", 0),
+    # director exit code never fixed,
+    # SEE https://github.com/ITISFoundation/osparc-simcore/issues/1466
+    ("director", 1),
+    ("director-v2", 0),
+    ("api-server", 0),
+    ("migration", 143),

```

Comment:
same here. why 143? what does it mean?

Reply:
I have no idea why this service, the migration which was crated by us exists with exit code 143 when rebooted.

Reply:
maybe @pcrespov ? I guess there must be something wrong with how the service is exiting. but that's not very important as long as it works as intended.

Reply:
Yes, the idea behind is that all exit codes are the ones being expected by these services.

Reply:
well I think the 143 in the migration service is not very expected... maybe at least add a FIXME for later.